### PR TITLE
[move-prover] Changes to bytecode instruction set to allow writeback to have edge information.

### PR DESCRIPTION
**Pull request by JakeSilverman**
## Motivation

Changes to bytecode instruction writeback to allow for edge information.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
## Test Plan

Current coverage is fine (as edge information not actually used yet).

### CODE REVIEW

File: language/move-prover/bytecode/src/stackless_bytecode.rs [hunk 1]
```diff
@@ -226,6 +226,38 @@ impl BorrowNode {
     }
 }
 
+/// A borrow edge with a known offset -- used in memory operations
```
Comment (wrwg) [inline language/move-prover/bytecode/src/stackless_bytecode.rs:229]: Please move the type declarations to the top of the file where the others are (Bytecode, Operation). Boilerplate impls (like for Display) should go towards the end,

**Review by wrwg — APPROVED** [review]
Please give the PR a descriptive title, and add `[move-prover]` before the title.

**Comment by JakeSilverman** [comment]
> Please give the PR a descriptive title, and add `[move-prover]` before the title.

Oops. Accidentally the PR title as branch name!

**Pull request merged**